include/enums.h

```diff
@@ -3044,6 +3044,29 @@ enum ShadingProfile {
     SHADING_MIM_09,
     SHADING_MIM_11,
     SHADING_MIM_12,
+    SHADING_ARN_09 = 0x00080000,
+    SHADING_ARN_10,
+    SHADING_ARN_11,
+    SHADING_ARN_12,
+    SHADING_ARN_13,
+    SHADING_DGB_06 = 0x00090000,
+    SHADING_KZN_01 = 0x000A0000,
+    SHADING_KZN_02,
+    SHADING_KZN_03,
+    SHADING_KZN_04,
+    SHADING_KZN_05,
+    SHADING_KZN_06,
+    SHADING_KZN_07,
+    SHADING_KZN_08,
+    SHADING_KZN_09,
+    SHADING_KZN_10,
+    SHADING_KZN_11,
+    SHADING_KZN_17,
+    SHADING_KZN_18,
+    SHADING_KZN_19,
+    SHADING_KZN_20,
+    SHADING_KZN_22,
+    SHADING_KZN_23,
 };
```

Comment:
Are any of these actually different?

Reply:
We don't split shading data for pal yet https://github.com/pmret/papermario/blob/5384f3b1638f615164007b3ea59c97f10c78e7c9/ver/pal/splat.yaml

Reply:
If the enum values are the same though, perhaps we should just put a TODO here re: splitting.

Reply:
unless im misunderstanding, [there is](https://github.com/pmret/papermario/blob/5384f3b1638f615164007b3ea59c97f10c78e7c9/include/enums.h#L2980)

Reply:
I meant if the enum is not different, why have it conditionally defined based on version at this time? It may make sense to eventually do so with includes once they are all split, but right now it seems to suggest there is a version-specific difference when there isn't.